### DISCUSSION TITLE
Any docs on how to add an animation to a vuejs project?

### DISCUSSION THREAD

**Discussion by dinchu** [motion-canvas/motion-canvas#1014] [Q&A]
I tried the player but the compiled js file doesn't work, however i would be more interested in writing animations inside my vuejs app with the editor

**Comment by dinchu** [comment]
@aarthificial any short tutorial on how to generate properly the js file that is added to the videoplayer?
i create a new project in motion canvas and run the build comand, that generates a single .js in the dist folder, in another project i add the canvas-player with the src pointing to my file an i get js errors.

![image](https://github.com/motion-canvas/motion-canvas/assets/5710370/6fff90fe-8dc2-4342-84e8-3f71d5546633)


I tried copying the animation js files from the docs (EG: https://motioncanvas.io/examples/tweening-linear.js) , and that renders an animation in the player without issues. What is the difference? how do you compile those?
![image](https://github.com/motion-canvas/motion-canvas/assets/5710370/5a7ac3c3-9743-477a-993f-d417849da4f7)

**Reply by KitenCoder** [reply]
Hello.
I want to ask a question about the motion-canvas library.
I couldn't find the author's email address.
Maybe you can answer me or write where I can contact you.
If you load https://motioncanvas.io/blog/,
then the player and editor are displayed there.
In the editor, you can correct the code and press a special button.
After that, the changes are displayed in the player.
Where can I read the instructions on how to insert such a player and editor
on my html page?
I would be very grateful for any advice.
